### DISCUSSION TITLE
Normalized cuts

### DISCUSSION THREAD

**Discussion by chenzhaiyu** [networkx/networkx#4670] [Algorithms]
Hi, I'm looking for a [normalized minimum cut ](https://people.eecs.berkeley.edu/~malik/papers/SM-ncut.pdf) implementation. Does networkx have a function to do that? The closest name I cloud find is `networkx.algorithms.cuts.normalized_cut_size`.

**Comment by dschult** [comment]
Many cut problems [can be mapped onto flow problems](https://networkx.org/documentation/stable//reference/algorithms/flow.html). 
This include the function `nx.minimum_cut` which uses weighted edges to determine minimum cut.
But I don't think the minimum is normalized.  The edge weight is arbitrary so can you normalize the edge weights to obtain a normalized cut size?  I'm not sure.  Anyway, there might be a way to get it using this code.
